design/ceph/object/store.md

```diff
@@ -351,3 +351,285 @@ spec:
+metadata:
+  name: hot.rgw.buckets.index
+  namespace: rook-ceph
+spec:
+  ...
+---
+# Bucket Data pool for cold storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: cold.rgw.buckets.data
+  namespace: rook-ceph 
+spec:
+  ...
+---
+# Bucket index pool for cold storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: cold.rgw.buckets.index
+  namespace: rook-ceph
+spec:
+  ...
+```
+
+If needed, create additional pool to use it in custom StorageClass, for example `GLACIER` and `REDUCED_REDUNDANCY`:
+> [!NOTE]  
+> Ceph allows arbitrary name for StorageClasses, however some clients/libs insist on AWS names so it is recommended to use one of the [valid x-amz-storage-class values](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObject.html#API_PutObject_RequestSyntax) for better compatibility:
+>
+> `STANDARD | REDUCED_REDUNDANCY | STANDARD_IA | ONEZONE_IA | INTELLIGENT_TIERING | GLACIER | DEEP_ARCHIVE | OUTPOSTS | GLACIER_IR | SNOW | EXPRESS_ONEZONE`
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: glacier.rgw.buckets.data
+  namespace: rook-ceph 
+spec:
+  ...
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: reduced.rgw.buckets.data
+  namespace: rook-ceph 
+spec:
+  ...
+```
+
+Then update `CephObjectStore` CRD  `sharedPools` section with target placements and StorageClasses definitions:
+
+```yaml
+spec:
+  # [OPTIONAL] - sharedPools is mutually exclusive with spec.metadataPool and spec.dataPool
+  # It is the place to tell RGW which pools can be used to store buckets and objects data and metadata.
+  # All pools under this section should be created beforehand.
+  # Rook will append rados namespace to each of the pools, so it is safe to use the same pool across multiple RGW instances.
+  # It is also safe to use the same pool in different target placements.
+  sharedPools: 
+    #  [REQUIRED] - metadata pool. Will be used by default to store bucket index
+    metadataPoolName: default.rgw.buckets.index # results to default.rgw.buckets.index:<rgw_name>.index
+
+    #  [REQUIRED] - data pool. Will be used by default to store bucket objects
+    dataPoolName: default.rgw.buckets.data # results to default.rgw.buckets.data:<rgw_name>.data
+
+    #  [OPTIONAL] - non erasure coded data pool. Will be used by default to store bucket multipart objects
+    # If not set, dataPoolName will be used
+    dataNonECPoolName: "" # results to default.rgw.buckets.data:<rgw_name>.data-nonec
+
+    # [OPTIONAL] - Storage classes for default placement.
+    storageClasses: 
+      - name: GLACIER # [REQUIRED] - can be arbitrary but it is better to use one of AWS x-amz-storage-class names.
+        # [REQUIRED] overrides default dataPoolName for object with given StorageClass.
+        dataPoolName: glacier.rgw.buckets.data # results to glacier.rgw.buckets.data:<rgw_name>.glacier
+
+    # [OPTIONAL] - additional pool placement targets.
+    additional: 
```

Comment:
If the settings above are under `placement`, then it seems natural to set this to `additionalPlacements`.
```suggestion
    additionalPlacements: 
```